Update entity reference output

steveworley opened the pull request:
- Shows total number of referenced entities found
- Update db_prefix usage to prevent errors when not set it conf

steveworley committed:
Update entity reference output

- Shows total number of referenced entities found
- Update db_prefix usage to prevent errors when not set it conf

seanhamlin commented on src/Base/DrushCaller.php:
```diff
@@ -115,13 +115,10 @@ public function sqlQuery($sql) {
     if (is_null($this->db_prefix)) {
       $sql_conf = $this->sqlConf('--format=json')->parseJson();
-
+      $this->db_prefix = isset($sql_conf->prefix) ? $sql_conf->prefix : NULL;
```
I think it should be a blank string if not found. Setting it to NULL will force site-audit to check every database query to check the sql-conf again.

steveworley committed:
Update Reference Autocomplete to use serializer

- Adds serializer which will atempt to repair broken serialised strings
- Updates entity ref to use the serializer

steveworley committed:
Fixed code review

- Assigns empty string to db_prefix if not found.
